### DISCUSSION TITLE
How do i leave a comunity?

### DISCUSSION THREAD

**Discussion by silvaf6** [stoatchat/discussions#643] [Q&A] [answered]
After 2 hours trying to share my community link, now I'm listed on the oficial one (i guess) and some bot community. Already lost alot of time and can't find a solution.
Thanks

**Comment by Rexogamer** [comment, marked as answer]
You seem to be asking two questions here? The steps for both depend on your platform:
- On desktop: to leave a server, right click on its icon and select "Leave server" - to create an invite, right click on the server's icon or an individual channel, then select "Create invite"
- On mobile: Long tapping in the same places *should* work but it is a bit flaky in my experience
